docs_src/design/adr/0006-Metrics-Collection.md

```diff
@@ -28,7 +28,7 @@ This ADR outlines ** metrics (or telemetry) ** collection and handling.
 
 ## Context
 
-[System Management services](../../microservices/system-management/Ch_SystemManagement.md) (SMA and executors) currently provide a limited set of “metrics” to requesting clients (3rd party applications and systems external to EdgeX).  Namely, it provides requesting clients with service CPU and memory usage; both metrics about the resource utilization of the service (the executable) itself versus metrics that are about what is happening inside of the service.  Arguably, the current system management metrics can be provided by the container engine and orchestration tools (example: by Docker engine) or by the underlying OS tooling.
+[System Management services](about:blank) (SMA and executors) currently provide a limited set of “metrics” to requesting clients (3rd party applications and systems external to EdgeX).  Namely, it provides requesting clients with service CPU and memory usage; both metrics about the resource utilization of the service (the executable) itself versus metrics that are about what is happening inside of the service.  Arguably, the current system management metrics can be provided by the container engine and orchestration tools (example: by Docker engine) or by the underlying OS tooling.

```

Comment:
This is an ADR which should not be modified.

Reply:
But it references documentation that is EOL.   If I leave the reference in, the build will fail on the dead link to `../../microservices/system-management/Ch_SystemManagement.md`.   If I leave `microservices/system-management/executor/Ch_SysMgmtExecutor.md` then the build will fail on the dead link to `https://github.com/edgexfoundry/edgex-go/blob/master/internal/system/executor/commands.go`

What to do?

Reply:
Ahh, got it. That works.